test/stdlib/MultilineString.swift

```diff
@@ -54,9 +54,9 @@ print(delimit("""
 // CHECK: -5-
 print("-5-")
 // CHECK-NEXT: {{^}}<Five
-// CHECK-NEXT: {{^}}
-// CHECK-NEXT: {{^}}
-// CHECK-NEXT: {{^}}Epsilon>
+// CHECK: {{^}}
+// CHECK: {{^}}
+// CHECK: {{^}}Epsilon>
```

Comment:
Otherwise I got this error:
```
test/stdlib/MultilineString.swift:57:16: error: CHECK-NEXT: is on the same line as previous match
// CHECK-NEXT: {{^}}
               ^
<stdin>:14:6: note: 'next' match was here
<Five
     ^
<stdin>:14:6: note: previous match ended here
<Five
     ^
```